.github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md

```diff
@@ -1,4 +1,18 @@
 ---
+name: "\U0001F525 Fireside Chat Checklist"
+about: Fireside Chat Planning Checklist
+title:
+labels:
+assignees: ''
+
+---
+
+**The upcoming Fireside chat is on Date - Month - Year**
+
+Register here to attend: [add registration page link].
+
+## Before the Event -- at least 6-8 weeks in advance
+---
 name: "\u1F525 Fireside Chat Checklist"
```

Comment:
```suggestion

```

This looks like it's duplicated from above? Ignore if not!

Reply:
It was duplicated!